CONTRIBUTORS

```diff
@@ -19,7 +19,7 @@ David Schneider
 Dennis Lockhart
 dimonf
 dumpweed
-eyqs
+Eugene Y. Q. Shen
```

Comment:
CONTRIBUTORS is an auto-generated file.  It didn't seem to grab your name properly from the git history, so the actual fix is to add a line to build/CONTRIBUTORS.sed to make the adjustment you did here.